src/DesignCompile/NetlistElaboration.cpp

```diff
@@ -1421,6 +1421,24 @@ bool NetlistElaboration::high_conn_(ModuleInstance* instance) {
       }
     }
   }
+  // Finally any unconnected ports with default value gets assigned the value
+  if (ports) {
+    for (auto p : *ports) {
+      if (p->High_conn() == nullptr) {
+        auto itr = allSignals.find(p->VpiName());
```

Comment:
Calling iterators that come back from a `find()` operation are much easier to understand if they are named `found` (which is what the represent) not `itr` (which is technically what the type is, but has absolutely no meaning in the context).

```c++
  auto found = allSignals.find(p->VpiName());
```